core/tests/README.md

```diff
@@ -0,0 +1,66 @@
+| `TESTS_LOCALE`                       | your store's default locale | The locale to use for tests if your Catalyst storefront supports multiple languages.                                                                                                                                                                                                                                                                                       |
+| `TESTS_FALLBACK_LOCALE`              | your store's default locale | If you are using an alternate locale for testing, this is the locale that will be used if a translation key cannot be found.                                                                                                                                                                                                                                               |
+| `TEST_CUSTOMER_ID`                   |                             | The ID of your dedicated testing customer account (if applicable). If you don't want tests to create multiple customer accounts, this is useful to ensure all tests are run against the same account. This is also useful if you are running tests in `READ_ONLY` mode. **NOTE: If this is false, each test that requires a customer session will create a new customer.** |
+| `TEST_CUSTOMER_EMAIL`                |                             | The email address of your dedicated testing customer account.                                                                                                                                                                                                                                                                                                              |
+| `TEST_CUSTOMER_PASSWORD`             |                             | The password for your dedicated testing customer account.                                                                                                                                                                                                                                                                                                                  |
+| `DEFAULT_PRODUCT_ID`                 |                             | The default product ID to use for tests that require a product. If this is not provided, any tests that require a product will fail.                                                                                                                                                                                                                                       |
```

Comment:
🍹 Historically when we've tried to maintain a table with detailed descriptions of environment variables, they easily become out of sync if someone updates `.env.test.example` and forgets to update this README. How do you feel about moving the descriptions to `.env.test.example` as comments, where defaults are already set?